test/sqllogictest/funcs.slt

```diff
@@ -288,6 +288,11 @@ SELECT ceil(1.1), ceil(1.111), ceil(100.1), ceil(100.11), ceil(-4.1)
 ----
 2  2  101  101  -4
 
+query RRRRR
```

Comment:
There are also existing (and commented out) tests for `ceiling` in `float8.slt` already: https://github.com/MaterializeInc/materialize/blob/b20c36cb926d6d88c0f797557eb92c496fa4bbd7/test/sqllogictest/postgres/float8.slt#L235

We should now uncomment these and remove the TODO.

Reply:
good catch. For my edification did you just grep for the term `ceiling` to find this?

Reply:
I did! :sweat_smile: